bsip-0048.md

```diff
@@ -10,45 +10,61 @@
+
+The ability to change the precision of a created asset adds flexibility in the asset creation process.
 
 # Rational
 
-The `disable_modify_max_supply` is a new created flag in the `asset_issuer_permissions`.
-The flag can be activated to forbid the modification of the `max_supply` (`asset_object.options`).
-Once the flag is activated, it can't be deactivated.
+The `disable_modify_max_supply` is a newly created flag in the `asset_issuer_permissions`. The flag can be activated to forbid the modification of the `max_supply` (`asset_object.options`). Once the flag is set to `true`, it can only be set to `false` if `current_supply` is zero.
+
+The `disable_issue` is a newly created flag in the `asset_issuer_permissions`. The flag can be activated to forbid issuance of additional tokens. Once the flag is set to `true`, it can not be changed.
+
+Changing the precision of an asset should be easily accomplished with an `asset_update_operation`. This causes no harm while the asset is not issued.
 
 # Specifications
 
-This BSIP comes with a change to the protocol, which requires a propotocol upgrade.
-However the changes to the code are minor.
+This BSIP comes with changes to the protocol, hence requires a propotocol upgrade. However the changes to the code are minor.
+
+For `disable_modify_max_supply`:
 
 1. Add the `disable_modify_max_supply` flag to the issuer permissions
-2. Add a `can_modify_max_supply()` the `asset_object`
-3. Change the `asset_update_evaluator::evaluate` method to assert false for
-    1. the deactivation of the `disable_modify_max_supply` flag
-    2. the modification of the `max_supply` when the `disable_modify_max_supply` is set
+2. Set `disable_modify_max_supply` to false for existing assets at hardfork time
+3. Add a `can_modify_max_supply()` to the `asset_object`
+4. Change the `asset_update_evaluator::do_evaluate` method to assert false for
+    1. the setting of the `disable_modify_max_supply` flag to `false` if `current_supply` is not zero or
+    2. the modification of the `max_supply` when the `disable_modify_max_supply` is `true`
+   
+For `disable_issue`:
+
+1. Add the `disable_issue` flag to the issuer permissions
+2. Set the `disable_issue` flag to `false` for existing assets at hardfork time
+3. Add a `can_issue()` to the `asset_object`
+4. Change the `asset_update_evaluator::do_evaluate` method to assert false for the setting of the `disable_issue` flag to `false` if it is currently `true`
+5. Change the `asset_issue_evaluator::do_evaluate` method to assert false if `can_modify_max_supply()` returns false
+
+To change the precision:
+
+1. Add the `fc::optional<uint8_t> new_precision` to the `asset_update_operation` struct.
+2. Change the `asset_update_evaluator::do_evaluate` method to assert false if `new_precision` exists and is not equal to `asset_object.precision` and `current_supply` is not zero.
   
```

Comment:
Please add a "Discussion" section that explains why the new flags have negative meaning.

Please add a "Risks" section that highlights the dangers of changing precision.